doc/adr/0013-format-of-eid-client-url.md

```diff
@@ -0,0 +1,35 @@
+# 13. Format of the eID-Client URL
+
+Date: 2022-10-18
+
+## Status
+
+Accepted
```

Comment:
Supercedes `0006-url-format-in-qr-code.md`? Sure, that one does mention the URL format for the *qr code*, but still it's the only URL format adr we have so far.

Reply:
I thought about that as well. I decided that ADR `0012-mvp-mobile-only.md` supersedes `0006-url-format-in-qr-code.md` and therefore not this one. Or do you think both should supersede it?

Reply:
I think it could be a good way to link to the "old" behaviour.

And I saw in other ADRs, that the superceded ADRs get a link to the new one, so that one can directly see, that it's not valid anymore. Maybe you can add that?

Reply:
Good point. I will add both.

Reply:
Example: https://github.com/digitalservicebund/useid-architecture/blob/main/doc/adr/0002-flutter-as-framework-for-mobile-app-development.md

Reply:
Done